### DISCUSSION TITLE
Task Delivery

### DISCUSSION THREAD

**Discussion by elusive-ttv** [gameserverapp/Platform#98] [Suggestions]
I would love for there to be a 'Monthly' option for tasks, such as tier supporter token deliveries. All of our tier subs have monthly perks that we are having a hard time transferring over to GSA with only x minute, hour at x time, daily and weekly delivery options.

**Comment by elusive-ttv** [comment]
![taskdelivery](https://user-images.githubusercontent.com/100104764/155822903-1d3762d9-e1ff-4840-a458-9d94b1191532.png)

**Comment by mvaessen** [comment]
created development item #103
